src/napari_spatialdata/utils/_test_utils.py

```diff
@@ -52,3 +58,40 @@ def click_list_widget_item(qtbot: QtBot, widget: QListWidget, position: QPoint,
             Qt.KeyboardModifier.NoModifier,
             pos=position,
         )
+
+
+def take_screenshot(viewer: napari.Viewer, canvas_only: bool = False) -> NDArrayA | Any:
+    """Take screenshot of the Napari viewer.
+
+    Parameters
+    ----------
+    viewer
+        Instance of napari Viewer.
+
+    canvas_only
+        If True, only the canvas is saved, not the viewer window.
+    """
+    logger.info("Taking screenshot of viewer")
+    # to distinguish between the black of the image background and the black of the napari background (now white)
+    # TODO (melonora): remove when napari allows for getting rid of margins.
+    viewer.theme = "light"
```

Comment:
To be sure to restore the old state it is better to do:
old_theme = viewer.theme
viewer.theme = "light"
# take screenshot
viewer.theme = old_theme